tests/ui/error-codes/E0116.stderr

```diff
@@ -1,8 +1,8 @@
 error[E0116]: cannot define inherent `impl` for a type outside of the crate where the type is defined
-  --> $DIR/E0116.rs:1:1
+  --> $DIR/E0116.rs:1:6
    |
 LL | impl Vec<u8> {}
-   | ^^^^^^^^^^^^^^^ impl for type defined outside of crate.
+   |      ^^^^^^^ impl for type defined outside of crate.
```

Comment:
I would prefer it if it also pointed at the `impl` keyword (`impl.start..ty.end`) to emphasize that the impl is wrong and not just something about the type like it being not found.

Reply:
how about now